mapss/templates/views/components/widgets/three-dimensional-widget.htm

```diff
@@ -48,9 +48,10 @@
         <div class="modal fade" data-bind="class: 'modal-' + $parent.unique_id + '-' + ko.unwrap($data.file_id)" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
             <div class="modal-dialog" role="document">
                 <div class="modal-content">
-                    <div class="modal-header">
+                    <div class="modal-header" style="display: flex; align-items: center;">
                     <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-                    <h4 class="modal-title" data-bind="text: title[$parent.activeLanguage].value"></h4>
+                    <h4 class="modal-title" style="padding-left: 15px" data-bind="text: title[$parent.activeLanguage].value"></h4>
+                    <button class="btn btn-lg btn-primary" data-bind="click: $parent.zoomToBounds" type="button" aria-label="Zoom to resource"><span aria-hidden="true">Zoom to Resource</span></button>
```

Comment:
I'd just remove the aria attributes here since we have a genuine label on the `<span>`